CHANGES.md

```diff
@@ -55,6 +55,8 @@
 - Vim plugin: prefix messages with `Black: ` so it's clear they come from Black (#3194)
 - Docker: changed to a /opt/venv installation + added to PATH to be available to
   non-root users (#3202)
+- GitHub Action: Updated usage of $GITHUB_ACTION_PATH to remove the nested python 
```

Comment:
I don't think a changelog entry is needed for this PR as this part should be totally transparent to users. I'll add a `skip news` label in the meanwhile.